clients/native/src/client/topology_control.rs

```diff
@@ -117,16 +117,16 @@ impl<T: NymTopology> TopologyAccessor<T> {
         self.inner.write().await.update(new_topology);
     }
 
-    pub(crate) async fn get_gateway_socket_url(&self, id: &str) -> Option<String> {
-        match &self.inner.read().await.0 {
-            None => None,
-            Some(ref topology) => topology
-                .gateways()
-                .iter()
-                .find(|gateway| gateway.pub_key == id)
-                .map(|gateway| gateway.client_listener.clone()),
-        }
-    }
+    // pub(crate) async fn get_gateway_socket_url(&self, id: &str) -> Option<String> {
+    //     match &self.inner.read().await.0 {
+    //         None => None,
+    //         Some(ref topology) => topology
+    //             .gateways()
+    //             .iter()
+    //             .find(|gateway| gateway.identity_key == id)
+    //             .map(|gateway| gateway.client_listener.clone()),
+    //     }
+    // }
```

Comment:
Is there a reason not to whack this commented code? It's always in source control if we want to go back to it, ja?

Reply:
The only reason I didn't whack it is because I forgot about it and it seems I didn't notice it before making the PR. It's gone now in my local branch.